napalm_logs/config/junos.yml

```diff
@@ -44,7 +44,7 @@ messages:
       variables: {}
       static:
         bgp//neighbors//neighbor//{peer}//state//session_state: CONNECT
-  - error: SNMP_TRAP_LINK_DOWN
+  - error: INTERFACE_DOWN
```

Comment:
The syslog message in cause seems to explicitly have the SNMP details (e.g. `snmpId` etc.). Isn't there any other possible syslog message that are not SNMP-related but pointing to the same event?